requirements-agent-release.txt

```diff
@@ -1,9 +1,9 @@
 datadog-active-directory==1.16.0; sys_platform == 'win32'
```

Comment:
Do I understand correctly that this file guarantees reproducible builds of the agent because the integration "deps" are all pinned?